docs/install/arm.md

```diff
@@ -51,7 +49,8 @@ curl https://raw.githubusercontent.com/prysmaticlabs/prysm/master/prysm.sh --out
 ./prysm.sh beacon-chain
 ```
 
-It is also recommended to include the `--p2p-host-ip` and `--min-sync-peers 7` flags to improve peering. For advanced users that desire standard debugging tools found in the Busybox base image, append a `--debug` flag to enable them.
+> Advanced users: Please refer to the article on how to [improve P2P connectivity](../p2p-host-ip).  
+Developers: Append `--debug` to enable standard debugging tools found in the Busybox base image.
```

Comment:
This needs to be removed; the debugging tools are only for Docker images, the new note is found https://github.com/prysmaticlabs/documentation/commit/e489ef1987e28da78aed1a74ecb8b32bdb96bd91

Reply:
removed.